bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ZA2JToZA2L2J_EWK_MLL50_Mjj120_5f_LO/ZA2JToZA2L2J_EWK_MLL50_Mjj120_5f_LO_run_card.dat

```diff
@@ -0,0 +1,270 @@
+  True    =  gridpack   !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  10000    =  nevents  ! Number of unweighted events requested 
+  0    =  iseed    ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+  1    =  lpp1     ! beam 1 type 
+  1    =  lpp2     ! beam 2 type
+  6800.0    =  ebeam1   ! beam 1 total energy in GeV
+  6800.0    =  ebeam2   ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+  0.0    =  polbeam1  ! beam polarization for beam 1
+  0.0    =  polbeam2  ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  lhapdf    =  pdlabel      ! PDF set                                     
+  $DEFAULT_PDF_SETS    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+  False    =  fixed_ren_scale   ! if .true. use fixed ren scale
+  False    =  fixed_fac_scale   ! if .true. use fixed fac scale
+  91.188    =  scale             ! fixed ren scale
+  91.188    =  dsqrt_q2fact1     ! fixed fact scale for pdf1
+  91.188    =  dsqrt_q2fact2     ! fixed fact scale for pdf2
+  1.0    =  scalefact         ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+  0    =  ickkw             ! 0 no matching, 1 MLM, 2 CKKW matching
+  1    =  highestmult       ! for ickkw=2, highest mult group
+  1    =  ktscheme          ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+  1.0    =  alpsfact          ! scale factor for QCD emission vx
+  False    =  chcluster         ! cluster only according to channel diag
+  True    =  pdfwgt            ! for ickkw=1, perform pdf reweighting
```

Comment:
This won't change anything to your sample but please keep this False so that people don't take this as a reference and make mistakes

Reply:
Hi @sihyunjeon,

Thanks for reviewing the cards.

We would prefer to keep the setting similar to the Run-2 ZGamma2JToGamma2L2J_EWK_MLL-50_MJJ-120_TuneCP5_13TeV-madgraph-pythia8 MC sample, because we are planning to do a Run-2 + Run-3 analysis.

The setting that were mentioned are all from the ZGamma2JToGamma2L2J_EWK_MLL-50_MJJ-120_TuneCP5_13TeV-madgraph-pythia8 MC sample.

Reply:
ok at least let's change pdfwgt, that doesn't make any difference